hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/handlers/NodeUpdateHandler.java

```diff
@@ -137,7 +139,10 @@ public Fees calculateFees(@NonNull final FeeContext feeContext) {
         return calculator.calculate();
     }
 
-    private Node.Builder updateNode(@NonNull final NodeUpdateTransactionBody op, @NonNull final Node node) {
+    private Node.Builder updateNode(
+            @NonNull final NodeUpdateTransactionBody op,
+            @NonNull final Node node,
+            @NonNull final NodesConfig nodeConfig) {
```

Comment:
We don't need `NodesConfig` injected here

Reply:
Ah yes, fixed!